example/appelagri/governance.ttl

```diff
@@ -0,0 +1,750 @@
+    metagov:hasNumber "1.1.1.1.1" ;
+    core:isPartOf govmeta:6cae72ef-fe12-4fc6-8b65-36dec4dd8e8a .    
+
+govmeta:1f01b3df-e8b3-4a31-b440-5c2be120173a a owl:NamedIndividual,   
+        metagov:Section ; 
+    core:hasTitle "Modèle de gouvernance"@fr ;
+    core:hasTitle "Governance Model"@en ;
+    core:hasTitle "Governance modell"@de ;
+    core:hasDescription "Il s'agit du cadre de règles, de processus et de procédures qui régissent l'acquisition, la gestion et l'utilisation des données au sein de la Zone."@fr ;
+    core:hasDescription "It refers to the framework of policies, processes, and procedures that govern the acquisition, management, and use of data within the Zone."@en ;
+    core:hasDescription "Er bezieht sich auf den Rahmen von Richtlinien, Prozessen und Verfahren, die den Erwerb, die Verwaltung und die Nutzung von Daten innerhalb der Zone regeln."@de ;
+    metagov:hasNumber "1.2" ; 
+    core:isPartOf govmeta:f16729ad-54b5-472c-8eec-7254bd78134a .    
+
+govmeta:07e574c6-5392-46ab-aca7-4772c4e098a7 a owl:NamedIndividual,   
+        metagov:Subsection ;
+    core:hasTitle "Créer des règles"@fr ; 
+    core:hasTitle "Create Rules"@en ; 
+    core:hasTitle "Regeln erstellen"@de ; 
+    core:hasDescription "Processus de création des règles et de procédures établies régissant la gestion et l'utilisation des ressources au sein de la Zone."@fr ;
+    core:hasDescription "Process of creating established policies and procedures governing the management and use of resources within the Zone."@en ;
+    core:hasDescription "Prozess der Erstellung von Richtlinien und Verfahren, die die Verwaltung und Nutzung von Ressourcen innerhalb der Zone regeln."@de ;
+    metagov:hasNumber "1.2.1" ;
+    core:isPartOf govmeta:1f01b3df-e8b3-4a31-b440-5c2be120173a .   
+
+govmeta:f6599a0e-67c9-44ae-88db-06fda92af4db a owl:NamedIndividual,   
+        metagov:Article ;
+    core:hasTitle "Création de la gouvernance par des utilisateurs spécifiques"@fr ;
+    core:hasTitle "Governance creation by Specific users"@en ;
+    core:hasTitle "Schaffung von Governance durch bestimmte Benutzer"@de ;
+    core:hasDescription "Clés ayant la permission de créer des règles relatives à la Zone."@fr ;
+    core:hasDescription "Keys with the permissions to create rules related to the Zone."@en ;
+    core:hasDescription "Schlüssel mit der Berechtigung, Regeln für die Zone zu erstellen."@de ;
+    metagov:hasNumber "1.2.1.1" ;
+    core:isPartOf govmeta:07e574c6-5392-46ab-aca7-4772c4e098a7 .   
+
+govmeta:1b7ef948-b319-4da2-94e3-7084add4ff35 a owl:NamedIndividual,   
+        metagov:Paragraph ;
+    core:hasTitle "Utilisateurs autorisés"@fr ;
+    core:hasTitle "Authorized users"@en ;
+    core:hasTitle "Autorisierte Benutzer"@de ;
+    core:hasDescription "did:key:0x04d1f1b8f8a7a28f9a5a254c326a963a22f5a5b5d5f5e5d5c5b5a5958575655"@en ;
+    core:hasDescription "did:key:7u9401g1b7g1a7a28f9a5a239c326a9813a22f5a5b5d5f5e5d5c5b5a478998568"@en ;
```

Comment:
There should be only one description per Paragraph for a considered language. Bascally a `Paragraph` has a single description, a free form expressed in a language (en, fr), which specifies the most elementary conditions that an article to which these rules relate must satisfy. Worth noting that the description supports markdown.

Reply:
I think it would be interesting to add a unit test to cover this requirement (cf: hello https://github.com/okp4/ontology/issues/103).